# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/unbreak-advansys-scsi.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Aside from tons of compiler warning, I have not found out via google, why the
AdvanSys driver is marked broken.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.16/drivers/scsi/Kconfig.orig	2006-05-03 19:50:59.113854500 +0200
+++ linux-2.6.16/drivers/scsi/Kconfig	2006-05-03 19:54:52.256425000 +0200
@@ -458,7 +458,7 @@
 	tristate "AdvanSys SCSI support"
 	depends on SCSI
 	depends on ISA || EISA || PCI
-	depends on BROKEN || X86_32
+	depends on X86_32
 	help
 	  This is a driver for all SCSI host adapters manufactured by
 	  AdvanSys. It is documented in the kernel source in
